Formula/netcdf.rb

```diff
@@ -0,0 +1,107 @@
+class Netcdf < Formula
+  desc "Libraries and data formats for array-oriented scientific data"
+  homepage "https://www.unidata.ucar.edu/software/netcdf"
+  url "ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-4.4.1.1.tar.gz"
+  mirror "https://www.gfd-dennou.org/library/netcdf/unidata-mirror/netcdf-4.4.1.1.tar.gz"
+  sha256 "4d44c6f4d02a8faf10ea619bfe1ba8224cd993024f4da12988c7465f663c8cae"
```

Comment:
revision needs to be >= 5 (probably 6 if we're going to change the fortran default)